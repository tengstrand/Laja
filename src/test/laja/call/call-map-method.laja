
#write "target/test-output/call-map-method.out"
  ## Put the variable 'b' into the default ($) namespace (that is a java.util.Map)
  #$.putAll({ b: "value" })
  #$.putAll( c: "123" )
$.b = {$.b}
$.c = {$.c}
#end
