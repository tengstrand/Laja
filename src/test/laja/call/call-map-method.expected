$.b = value
$.c = 123
